src/libint2_wrappers/libint2_wrappers.h

```diff
@@ -82,7 +82,7 @@ void parallel_do(Lambda& lambda, int nthreads) {
 }
 #else  // use C++11 threads
 std::vector<std::thread> threads;
-for (int thread_id = 0; thread_id != libint2::nthreads; ++thread_id) {
+for (int thread_id = 0; thread_id != nthreads; ++thread_id) {

```

Comment:
I have no idea does libint2 namespace have nthreads variable or not. I suppose it is better to take the value from functions argument

Reply:
i think using libint-namespace variables would probably be safer since this is something they use internally